src/engine/ExportQueryExecutionTrees.cpp

```diff
@@ -14,13 +14,8 @@
 
 // __________________________________________________________________________
 namespace {
-// Return a range that contains the indices of the rows that have to be exported
-// from the `idTable` given the `LimitOffsetClause`. It takes into account the
-// LIMIT, the OFFSET, and the actual size of the `idTable`
-auto getRowIndices(const LimitOffsetClause& limitOffset,
-                   const IdTable& idTable) {
-  return std::views::iota(limitOffset.actualOffset(idTable.size()),
-                          limitOffset.upperBound(idTable.size()));
+auto getRowIndices(const IdTable& idTable) {
+  return std::views::iota(0u, idTable.size());
```

Comment:
It might seem silly to return this trivial range now, just to iterate over all the rows, but it makes sense again when extended to iterate over multiple `IdTable`s in  #1350